hydradx/model/amm/omnipool_amm.py

```diff
@@ -1,6 +1,66 @@
 import copy
 import string
 
+import pytest
+
+
+def state_dict(
+        token_list: list[str],
+        r_values: list[float],
+        q_values: list[float] = None,
+        p_values: list[float] = None,
+        b_values: list[float] = None,
+        s_values: list[float] = None,
+        omega_values: list[float] = None,
+        L: float = 0,
+        D: float = 0,
+        fee_assets: float = 0.0,
+        fee_lrna: float = 0.0,
+        preferred_stablecoin: str = 'USD'
+) -> dict:
+    assert 'HDX' in token_list, 'HDX not included in token list'
+    assert len(r_values) == len(token_list), 'list lengths do not match'
+    # get initial value of T (total value locked)
+
+    if not omega_values:
+        omega_values = [0 for _ in range(len(token_list))]
```

Comment:
if no omegas (weight caps) are specified, shouldn't we default to 1? This means no asset can be more than 100% of the pool... so in effect, no cap.